### DISCUSSION TITLE
How To run Offline , ABP downloads File api.github.com/repos/abpframework/abp

### DISCUSSION THREAD

**Discussion by mrs01dev** [abpframework/abp#7314] [Q&A]
ABP framework do request  for  https://api.github.com/repos/abpframework/abp
with the content 
{
  "id": 75505125,
  "node_id": "MDEwOlJlcG9zaXRvcnk3NTUwNTEyNQ==",
  "name": "abp",
  "full_name": "abpframework/abp",
  "private": false,
  "owner": {
    "login": "abpframework",
    "id": 42068324,
    "node_id": "MDEyOk9yZ2FuaXphdGlvbjQyMDY4MzI0",
    "avatar_url": "https://avatars3.githubusercontent.com/u/42068324?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/abpframework",
    "html_url": "https://github.com/abpframework",
    "followers_url": "https://api.github.com/users/abpframework/followers",
    "following_url": "https://api.github.com/users/abpframework/following{/other_user}",
    "gists_url": "https://api.github.com/users/abpframework/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/abpframework/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/abpframework/subscriptions",
    "organizations_url": "https://api.github.com/users/abpframework/orgs",
    "repos_url": "https://api.github.com/users/abpframework/repos",
    "events_url": "https://api.github.com/users/abpframework/events{/privacy}",
    "received_events_url": "https://api.github.com/users/abpframework/received_events",
    "type": "Organization",
    "site_admin": false
  },
  "html_url": "https://github.com/abpframework/abp",
  "description": "Open Source Web Application Framework for ASP.NET Core",
  "fork": false,
  "url": "https://api.github.com/repos/abpframework/abp",
  "forks_url": "https://api.github.com/repos/abpframework/abp/forks",
  "keys_url": "https://api.github.com/repos/abpframework/abp/keys{/key_id}",
  "collaborators_url": "https://api.github.com/repos/abpframework/abp/collaborators{/collaborator}",
  "teams_url": "https://api.github.com/repos/abpframework/abp/teams",
  "hooks_url": "https://api.github.com/repos/abpframework/abp/hooks",
  "issue_events_url": "https://api.github.com/repos/abpframework/abp/issues/events{/number}",
  "events_url": "https://api.github.com/repos/abpframework/abp/events",
  "assignees_url": "https://api.github.com/repos/abpframework/abp/assignees{/user}",
  "branches_url": "https://api.github.com/repos/abpframework/abp/branches{/branch}",
  "tags_url": "https://api.github.com/repos/abpframework/abp/tags",
  "blobs_url": "https://api.github.com/repos/abpframework/abp/git/blobs{/sha}",
  "git_tags_url": "https://api.github.com/repos/abpframework/abp/git/tags{/sha}",
  "git_refs_url": "https://api.github.com/repos/abpframework/abp/git/refs{/sha}",
  "trees_url": "https://api.github.com/repos/abpframework/abp/git/trees{/sha}",
  "statuses_url": "https://api.github.com/repos/abpframework/abp/statuses/{sha}",
  "languages_url": "https://api.github.com/repos/abpframework/abp/languages",
  "stargazers_url": "https://api.github.com/repos/abpframework/abp/stargazers",
  "contributors_url": "https://api.github.com/repos/abpframework/abp/contributors",
  "subscribers_url": "https://api.github.com/repos/abpframework/abp/subscribers",
  "subscription_url": "https://api.github.com/repos/abpframework/abp/subscription",
  "commits_url": "https://api.github.com/repos/abpframework/abp/commits{/sha}",
  "git_commits_url": "https://api.github.com/repos/abpframework/abp/git/commits{/sha}",
  "comments_url": "https://api.github.com/repos/abpframework/abp/comments{/number}",
  "issue_comment_url": "https://api.github.com/repos/abpframework/abp/issues/comments{/number}",
  "contents_url": "https://api.github.com/repos/abpframework/abp/contents/{+path}",
  "compare_url": "https://api.github.com/repos/abpframework/abp/compare/{base}...{head}",
  "merges_url": "https://api.github.com/repos/abpframework/abp/merges",
  "archive_url": "https://api.github.com/repos/abpframework/abp/{archive_format}{/ref}",
  "downloads_url": "https://api.github.com/repos/abpframework/abp/downloads",
  "issues_url": "https://api.github.com/repos/abpframework/abp/issues{/number}",
  "pulls_url": "https://api.github.com/repos/abpframework/abp/pulls{/number}",
  "milestones_url": "https://api.github.com/repos/abpframework/abp/milestones{/number}",
  "notifications_url": "https://api.github.com/repos/abpframework/abp/notifications{?since,all,participating}",
  "labels_url": "https://api.github.com/repos/abpframework/abp/labels{/name}",
  "releases_url": "https://api.github.com/repos/abpframework/abp/releases{/id}",
  "deployments_url": "https://api.github.com/repos/abpframework/abp/deployments",
  "created_at": "2016-12-03T22:56:24Z",
  "updated_at": "2021-01-18T12:24:09Z",
  "pushed_at": "2021-01-18T13:44:31Z",
  "git_url": "git://github.com/abpframework/abp.git",
  "ssh_url": "git@github.com:abpframework/abp.git",
  "clone_url": "https://github.com/abpframework/abp.git",
  "svn_url": "https://github.com/abpframework/abp",
  "homepage": "https://abp.io",
  "size": 212319,
  "stargazers_count": 5469,
  "watchers_count": 5469,
  "language": "C#",
  "has_issues": true,
  "has_projects": false,
  "has_downloads": true,
  "has_wiki": false,
  "has_pages": false,
  "forks_count": 1681,
  "mirror_url": null,
  "archived": false,
  "disabled": false,
  "open_issues_count": 559,
  "license": {
    "key": "lgpl-3.0",
    "name": "GNU Lesser General Public License v3.0",
    "spdx_id": "LGPL-3.0",
    "url": "https://api.github.com/licenses/lgpl-3.0",
    "node_id": "MDc6TGljZW5zZTEy"
  },
  "forks": 1681,
  "open_issues": 559,
  "watchers": 5469,
  "default_branch": "dev",
  "temp_clone_token": null,
  "organization": {
    "login": "abpframework",
    "id": 42068324,
    "node_id": "MDEyOk9yZ2FuaXphdGlvbjQyMDY4MzI0",
    "avatar_url": "https://avatars3.githubusercontent.com/u/42068324?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/abpframework",
    "html_url": "https://github.com/abpframework",
    "followers_url": "https://api.github.com/users/abpframework/followers",
    "following_url": "https://api.github.com/users/abpframework/following{/other_user}",
    "gists_url": "https://api.github.com/users/abpframework/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/abpframework/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/abpframework/subscriptions",
    "organizations_url": "https://api.github.com/users/abpframework/orgs",
    "repos_url": "https://api.github.com/users/abpframework/repos",
    "events_url": "https://api.github.com/users/abpframework/events{/privacy}",
    "received_events_url": "https://api.github.com/users/abpframework/received_events",
    "type": "Organization",
    "site_admin": false
  },
  "network_count": 1681,
  "subscribers_count": 314
}
Is this content important ?
How to work offline without downloading this file ?

**Comment by heitorgiacomini** [comment]
I want to disable it.
